src/frequenz/client/common/microgrid/components/components.py

```diff
@@ -0,0 +1,74 @@
+# License: MIT
+# Copyright © 2022 Frequenz Energy-as-a-Service GmbH
+
+"""Defines the components that can be used in a microgrid."""
+from __future__ import annotations
+
+from enum import Enum
+
+# pylint: disable=no-name-in-module
+from frequenz.api.common.v1.microgrid.components.components_pb2 import (
+    ComponentCategory as PBComponentCategory,
+)
+
+# pylint: enable=no-name-in-module
+
+
+class ComponentCategory(Enum):
+    """Possible types of microgrid component."""
+
+    NONE = PBComponentCategory.COMPONENT_CATEGORY_UNSPECIFIED
+    """Unspecified component category."""
```

Comment:
Same about the documentation of all these items, we should use the same as in the protobuf stuff, or at least that at the minimum/base, maybe we can extend it with python-specific stuff too (or remove stuff that it is too protobuf-specific).